storybook/storybook-react/src/components/OrderedList/OrderedList.docs.mdx

```diff
@@ -1,15 +1,52 @@
-import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import { Canvas, Meta } from "@storybook/blocks";
 import * as OrderedListStories from "./OrderedList.stories.tsx";
-import README from "../../../../../packages/css/src/ordered-list/README.md?raw";
 
 <Meta of={OrderedListStories} />
 
-<Markdown>{README}</Markdown>
+# Ordered list
 
-<Primary />
+Een geordende lijst is een verticale groep gerelateerde inhoud met een volgorde.
+
+## Voorbeelden
+
+### Basis
+
+De lijst heeft oplopende getallen als opsommingstekens.
+Er is genoeg ruimte voor getallen tot 99.
+Tussen de items staat extra witruimte.
+Dat geeft een beter onderscheid tussen de items, zeker als ze uit meerdere regels tekst bestaan.
+
+<Canvas of={OrderedListStories.Basis} />
```

Comment:
The controls are missing here

Reply:
I think we should offer controls on the individual story pages only, to keep the docs page easier to read. That’s also why I’m initially hiding the code now.

Reply:
Ah, I kinda disagree on that one, but let's discuss that in the seperate Storybook docs ticket